### DISCUSSION TITLE
Get plates/members from IFCcurtainWall

### DISCUSSION THREAD

**Discussion by jurvanoerle** [IfcOpenShell/IfcOpenShell#4491] [Q&A] [answered]
Hi ,
I want to get the members and plates representing an IFCcurtainwall (that has no representation itself). 

I tried the same procedure suggested for retreiving openings in a wall, but with no succes.
I also tried to follow the IFC definition, but helas, no succes. 

Any help is appreciated,
regards, Jur

**Comment by aothms** [comment]
Openings use IfcRelVoidsElements, the curtain wall to elements relationship is IfcRelAggregates, which is accessible under the inverse attribute name `IsDecomposedBy` when navigating from the parent.

**Comment by Moult** [comment]
Ifcopenshell.util.element has a get_decomposition function which you can use as a one liner.

Dion Moult

Sent from Proton Mail mobile

-------- Original Message --------

**Comment by jurvanoerle** [comment]
Hi,
Thanks for the swift reply, but could you supply some sample code. I dont know how to use get_decomposition or icfaggregates

**Reply by Moult** [reply, marked as answer]
```
import ifcopenshell.util.element
ifcopenshell.util.element.get_decomposition(curtain_wall)
```
